src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperDescriptionFactory.cs

```diff
@@ -207,6 +208,7 @@ internal static string CleanSummaryContent(string summaryContent)
                     summaryBuilder.Insert(cref.Index, $"`{reducedValue}`");
                 }
             }
+            summaryBuilder.Trim();
             var lines = summaryBuilder.ToString().Split(new[] { '\n' }, StringSplitOptions.None).Select(line => line.Trim());
```

Comment:
Why can't we just do a string trim here?

Reply:
Ya seems like a lot to do the trim on the string builder when we're already converting this thing down into a string.

Reply:
I suppose it IS a bit of a premature optimization. I'll ax the StringBuilder extensions and rock it old-school. If we find that this area is a hotbed for allocation we'll probably have to rework this whole section anyway.